actions/install-report-tools/README.md

```diff
@@ -0,0 +1,13 @@
+# install-droute
+
+This action will install droute for use when communicating with ReportPortal.
+
+## Example usage
+
+An example usage of this action from an external repository:
+
+```yaml
+steps:
+  - name: "Install droute tool"
+    uses: neuralmagic/nm-actions/actions/install-droute@main
```

Comment:
```suggestion
    uses: neuralmagic/nm-actions/actions/reportportal_submit_execution_results@main
```

Reply:
On the presumption that this README should be moved to the `reportportal_submit_execution_results` folder, I agree, but I also think this README doesn't really serve much purpose without documenting inputs & outputs, so overall I think either drop the README or add the useful bits (in addition to moving to the correct folder).

Reply:
This whole action is irrelevant now.  I think Dan just missed removing the folder and README when she removed the action.yml file.  That was done because the rh-reporter runner has the droute software installed already.
I'll delete this folder